app/src/main/java/com/vultisig/wallet/ui/models/send/SendFormViewModel.kt

```diff
@@ -187,6 +187,26 @@ internal class SendFormViewModel @Inject constructor(
 
         tokenAmountFieldState.setTextAndPlaceCursorAtEnd(max)
     }
+    fun choosePercentAmount(percent: Int) {
+        val selectedAccount = selectedAccount ?: return
+        val selectedTokenValue = selectedAccount.tokenValue ?: return
+        val gasFee = gasFee.value ?: return
+        val percentValue = if (selectedAccount.token.isNativeToken) {
+            val calculatedValue = maxOf(
+                BigInteger.ZERO,
+                (selectedTokenValue.value * percent.toBigInteger()) / BigInteger.valueOf(100) - gasFee.value
+            )
+            TokenValue(
+                value = calculatedValue,
+                unit = selectedTokenValue.unit,
+                decimals = selectedTokenValue.decimals,
+            )
+        } else {
+            selectedTokenValue
+        }.decimal.toPlainString()
```

Comment:
Because chooseMaxTokenAmount used this way https://github.com/vultisig/vultisig-android/blob/03c4fac511a38f123d1fa2b7f7c3747c427e3a43/app/src/main/java/com/vultisig/wallet/ui/models/send/SendFormViewModel.kt#L173-L186

Reply:
Done